### ISSUE TITLE
Use new API to open in Browser

### ISSUE THREAD

**Issue by philosowaffle** [philosowaffle/vs-openapi-designer#28]
Currently using the `open` module to open the swagger doc in the browser. In the latest VS Code release (1.31.1) a new API is exposed that will do this for us.

```
await vscode.env.openExternal(vscode.Uri.parse("https://github.com/Microsoft/vscode/issues/66741"));
```

- Remove the dependency on `open`
- Use new api instead.

Also resolve this:
```
Extension 'philosowaffle.openapi-designer' uses the 'vscode.previewHtml' command which is deprecated and will be removed. Please update your extension to use the Webview API: https://go.microsoft.com/fwlink/?linkid=2039309
```

**Comment by philosowaffle** [comment]
https://code.visualstudio.com/api/extension-guides/webview
https://github.com/Microsoft/vscode-extension-samples/blob/master/webview-sample/src/extension.ts
https://github.com/arjun-g/vs-swagger-viewer/blob/master/src/preview/client.ts

**Comment by damccull** [comment]
Does this cover the fact that the preview and preview in side-bar aren't working, but preview in browser is?

**Comment by philosowaffle** [comment]
Probably. This extension depended on an API that was deprecated and removed in recent releases of VS Code.  This issue is to uplift and restore behavior.

**Comment by philosowaffle** [comment]
Fixed as part of v0.3.0 release.
